ntbs-build/openshift-pipelines/persistent-volume-claims/ntbs-build-pvc.yml

```diff
@@ -0,0 +1,14 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: ntbs-build-pvc
+  namespace: ntbs-build
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  volumeName: pvc-3413dc10-494a-49c0-915d-f99a7e9fb965
+  storageClassName: standard
+  volumeMode: Filesystem
```

Comment:
What is this used for? I can't quite work it out

Reply:
When the pipeline runs it requires a filesystem so that the files which are checked out in the clone step are still available in the deploy step. This needs to be selected manually when you trigger the pipeline.

Here we are setting up a persistent volume claim so that it can be used for this purpose. I was in two minds as to whether to commit this file and I'm still not sure - I decided that it's worth documenting as something that we need to create in order to be able to run the pipeline.

Reply:
I see, thanks for explaining that!